classes/asserters/phpResource.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace mageekguy\atoum\asserters;
+
+use
+	mageekguy\atoum\asserters
+;
+
+class phpResource extends asserters\variable
+{
+	public function setWith($value)
+	{
+		parent::setWith($value);
+
+		if ($this->analyzer->isResource($this->value) === true)
+		{
+			$this->pass();
+		}
+		else
+		{
+			$this->fail($this->_('%s is not a resource', $this));
+		}
+
+		return $this;
+	}
+
+	public function isOfType($type, $failMessage = null)
```

Comment:
I would add some helper assertions here like `isStream`, `isMysql`, ... and a wildcard helper via `__get/__call` (`is*`) to allow things like `->isDomXml`, `->isWhatever`, ...